samples/client/petstore/spring-cloud/src/main/java/io/swagger/model/Pet.java

```diff
@@ -13,10 +13,14 @@
 
 
 
+/**
+ * A pet for sale in the pet store
+ **/
 
 /**
- * Pet
+ * A pet for sale in the pet store
  */
```

Comment:
I don't see duplicated comments. Can you share a URL to the lines of code with duplicated comments in the latest master?